src/sidebar/components/test/annotation-test.js

```diff
@@ -1068,22 +1068,57 @@ describe('annotation', function() {
     });
 
     describe('tag display', function () {
-      it('displays links to tags on the stream', function () {
+      beforeEach('make serviceUrl() return a URL for the tag', function() {
         fakeServiceUrl
           .withArgs('search.tag', {tag: 'atag'})
           .returns('https://test.hypothes.is/stream?q=tag:atag');
+      });
 
-        var directive = createDirective(Object.assign(fixtures.defaultAnnotation(), {
+      /*
+       * Return an annotation directive with a single tag.
+       */
+      function annotationWithOneTag() {
+        return createDirective(Object.assign(fixtures.defaultAnnotation(), {
           tags: ['atag'],
         }));
+      }
+
+      /**
+       * Return the one tag link element from the given annotation directive.
+       */
+      function tagLinkFrom(directive) {
         var links = [].slice.apply(directive.element[0].querySelectorAll('a'));
         var tagLinks = links.filter(function (link) {
           return link.textContent === 'atag';
         });
-
         assert.equal(tagLinks.length, 1);
-        assert.equal(tagLinks[0].href,
-                     'https://test.hypothes.is/stream?q=tag:atag');
+        return tagLinks[0];
+      }
+
+      context('when the annotation is first-party', function() {
+        beforeEach('configure a first-party annotation', function() {
+          fakeAccountID.isThirdPartyUser.returns(false);
+        });
+
+        it('displays links to tags on the stream', function () {
```

Comment:
The client links to search pages for tags rather than the stream now. See https://hypothes.is/api/links